conf/default.toml

```diff
@@ -72,22 +72,22 @@ proxy_path = ""
 #  - for font families: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family
 #  - for background: https://developer.mozilla.org/en-US/docs/Web/CSS/background
 [theme]
-primary_color = "#0f4395"
-secondary_color = "#8bc832"
+primary_color = "#c82850"
+secondary_color = "#001638"
 main_color = "#212029" # All-purpose text color
 background_color = "#fdfbff"
 
 # This optional parameter indicates which background should be used for the main header
-# header_background = "center url('assets/img/default_header_bg.webp')"
+header_background = "center /cover url('assets/img/header_bg.webp')"
 
 # This optional parameter allows to override the fallback image that should be used for thumbnails, 
 # if the metadata record has no thumbnail image url or it is broken.
 # thumbnail_placeholder = 'assets/img/my_custom_placeholder.png'
 
 # These optional parameters allow changing fonts used in the app
-# main_font = "'My Custom Font', fallback-font"
-# title_font = "'My Custom Title Font', fallback-font-title"
-# fonts_stylesheet_url = "https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Permanent+Marker&display=swap"
+main_font = "'Rubik', sans-serif"
```

Comment:
I don't think these parameters should be set in the default config; once someone deploys the datahub, we will never be able to change the default values for them since they will be written in their config file;

This is why most settings are not set in the default config,